### ISSUE TITLE
NameError: name 'message' is not defined in nativemessaging.py get_message()

### ISSUE THREAD

**Issue by IberAI** [roelderickx/nativemessaging-ng#2]
When running `./test.sh`, the following error occurs:

```
Traceback (most recent call last):
  File "/home/iber/Development/nativeMessaging/src/nativemessaging-ng/test/test.py", line 7, in <module>
    msg = nativemessaging.get_message()
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/iber/Development/nativeMessaging/src/nativemessaging-ng/nativemessaging/nativemessaging.py", line 43, in get_message
    if message:
       ^^^^^^^
NameError: name 'message' is not defined
```

### **Expected Output:**
```
received msg: {'hello': 'world', 'test': 'example'}
"test"
```

### **Steps to Reproduce:**
1. Clone the repository and navigate to the `nativemessaging-ng/test`.
2. Run the test script using:
   ```bash
   ./test.sh
   ```

### **Actual Behavior:**
- The script throws a `NameError: name 'message' is not defined` on line 43 in `nativemessaging.py`.

### **Possible Cause:**
It seems that the variable `message` might not be defined or initialized properly before being referenced in `get_message()` function.  `message` should be the returned value of `get_message_raw()`.

### **Environment:**
- OS: Arch Linux
- Python version: [3.12.7]

**Comment by roelderickx** [comment]
Hi,

Sorry for the late reaction, I didn't receive a mail about this issue. That's as strange as the fact  I missed the call to get_message_raw, you're totally right about that.
Thanks for reporting, I'll create a new pip release in the evening.

**Comment by roelderickx** [comment]
The package is released as version 1.3.2
